Update documentation

jakeseawell opened the pull request.

jakeseawell committed:
magnite-streaming

 - Updated magnite-streaming.md
 - Updated audience-field-mapping image
 - Added new audience-target-dropdown image

jakeseawell committed:
magnite-streaming

 - Updated doc wording

jakeseawell committed:
magnite-streaming

 - Updated page title

jakeseawell committed:
magnite-batch

 - Updated doc and images

jakeseawell committed:
magnite-streaming

 - Updated doc

jakeseawell committed:
magnite-streaming

 - Fixed 'magnite-streaming' typos

jakeseawell committed:
magnite-streaming

 - Change 'Magnite Streaming' to 'Magnite'

jakeseawell committed:
magnite-batch

 - Updated 'Magnite Streaming' to 'Magnite'

jakeseawell committed:
magnite-batch

 - Added missing DNL

jakeseawell committed:
magnite-batch

 - Updated wording

jakeseawell committed:
magnite-batch

 - Updated 'Adobe Real-Time CDP audeinces' to 'Adobe CDP audeinces'

jakeseawell committed:
magnite-streaming

 - Updated magnite.streaming.md doc
 - Updated the screenshots for audience-field-mapping and audience-target-dropdown

jakeseawell committed:
magnite-batch

 - Updated magnite-batch.md doc
 - Updated the screenshots for audience-field-mapping and audience-select-device-type

jakeseawell committed:
Merge branch 'magnite-streaming' into magnite-batch

vfalon commented on help/destinations/catalog/advertising/magnite-streaming.md:
```diff
@@ -1,59 +1,57 @@
-| device_id         | A unique identifier for a device or identity. We accept any device ID and first-party ID regardless of type.           | Identity types supported by Magnite include but are not limited to PPUID, GAID, IDFA, and TV Device IDs.     | 
-
-{style="table-layout:auto"}
+The [!DNL Magnite: Real-Time] destination supports the activation of any attributes and/or identities stored in [!DNL Adobe Experience Platform], which will need to be mapped to a [!DNL Magnite] supported device type. The list of supported device types is managed by [!DNL Magnite], and may change and/or grow over time. The current list of supported device types will be available from a dropdown i ...
```
"The current list of supported device types will be available from a dropdown in the Target Field,".

There is no dropdown when you configure mapping. Please rephrase this part so it reflects the UI.

jakeseawell replied:
Will update `dropdown` to `'Select target field' popout window`

vfalon commented on help/destinations/catalog/advertising/magnite-streaming.md:
```diff
@@ -122,23 +120,33 @@ Once the destination connection has been created, you can proceed to the audienc
+The next step is mapping Source Field identifiers to the [!DNL Magnite] Device Type identifier of your choice, located in the Target Field.
 
-* You can add as many mappings as you need by selecting **[!UICONTROL Add new mapping]**.
+The current list of [!DNL Magnite] supported device types will be available via a dropdown list, by clicking the Target Field's target attribute button.
```
"The current list of [!DNL Magnite] supported device types will be available via a dropdown list, by clicking the Target Field's target attribute button.".

There is no dropdown when you configure mapping. Please rephrase this part so it reflects the UI.

jakeseawell replied:
Will update `dropdown` to `'Select target field' popout window`

vfalon replied:
Don't forget about this update please.

vfalon commented on help/destinations/catalog/advertising/magnite-streaming.md:
```diff
@@ -122,23 +120,33 @@ Once the destination connection has been created, you can proceed to the audienc
+The current list of [!DNL Magnite] supported device types will be available via a dropdown list, by clicking the Target Field's target attribute button.
 
-This example using the Real-Time destination shows a row that contains a generic deviceId source identifier mapped to the Magnite device_id target field. When you're with the mappings, select [!UICONTROL Next].
+In order to support a dynamic list of device types, the dropdown will provide a list of integers (from 0-50), each of which represents a specific device type in [!DNL Magnite]'s system. To understand which device type id in the list of integers corresponds to which [!DNL Magnite] device type, reach out to your [!DNL Magnite] account representative. 
```
Multiple issues here:

- he dropdown will provide a list of integers (from 0-50): there is no dropdown in any of the mapping configuration steps. Please rephrase so it reflects the UI.
- Is there some Magnite documentation where these integers are explained? Or is this a custom implementation with no public info? Having to reach out to Magnite whenever a user doesn't know what to select is not a great experience.

jakeseawell replied:
- Will update `dropdown` to `'Select target field' popout window`
- Yes, the former: There is a custom implementation with no public info. If the client does not know which device type to use for this stage, they should be reaching out to Magnite for more information
    - I have added that 0 (zero) should be considered the "default"

vfalon commented on help/destinations/catalog/advertising/magnite-batch.md:
```diff
@@ -1,25 +1,29 @@
+>The [!DNL Magnite] destinations can be used for ingestion into any [!DNL Magnite] platform: Streaming, DV+, SpringServe, etc.
 
-1. If you only want and/or need to deliver audiences once per day, you can use the Magnite: Batch destination, which delivers audiences to Magnite Streaming via a daily S3 batch file delivery. These Batch audiences are stored indefinitely in the Magnite platform, unlike real-time audiences, which are only stored for a couple days.
+Adobe CDP audiences can be delivered to the [!DNL Magnite] platform in two ways - they can be delivered once per day, or they can be delivered in real-time:
```
Please update all "Adobe CDP" occurences to "Adobe Real-Time CDP". Same for "CDP".

jakeseawell replied:
Can you clarify - is this an update to the naming of the `Adobe CDP` product -> `Adobe Real-Time CDP`? The reason I ask, is because Magnite has 2 Adobe-CDP destinations - 1 Real-Time destination, and 1 Batch destination. This file represents the document for the Magnite: Batch destination, so I worry about the use of the term "Real-Time" in this context. However, if the name change is required because the Adobe-CDP is now named Adobe Real-Time CDP, then I can make that change 👍

vfalon replied:
@jakeseawell Any time you refer to Adobe's CDP, the official branding is "Real-Time CDP". It has nothing to do with the type of destination platform. Please update all occurrences accordingly.

vfalon commented on help/destinations/catalog/advertising/magnite-batch.md:
```diff
@@ -147,12 +139,23 @@ In the **[!UICONTROL Source field]**, you can select any attribute or identity f
 >[!NOTE]
 >
->If you plan to send/map multiple ID types (GAID, IDFA, etc.) using the Batch destination, a new destination connection/instance is required for each. Please contact your Magnite Account representative for more information.
+>If you plan to map an audience to multiple device types (GAID, IDFA, etc.) using the Batch destination, a new destination connection/instance is required for each mapping. Please contact your [!DNL Magnite] account representative for more information.
```
"Please contact your [!DNL Magnite] account representative for more information."

What would they contact Magnite for? Is there anything else to do aside from creating a new connection? Anything that Magnite needs to do on their side?

jakeseawell replied:
This comment is specific to mapping multiple device types in a single destination instance. If they want to be able to map multiple device types, or they want/need more information about that, then they would need to reach out to Magnite for more information so we can advise them.

vfalon commented on help/destinations/catalog/advertising/magnite-batch.md:
```diff
@@ -161,19 +164,19 @@ On the "Configure a filename and export schedule for each audience" screen, you
 >
 > A Mapping ID or "NONE" is required for this destination.
 >
-> A Mapping ID should be provided when an audience has a pre-existing Segment ID previously known to Magnite Streaming. Otherwise, "NONE" should be used as the Mapping ID.
```
Please illustrate this with some screenshots. One with an existing mapping and one with NONE.

jakeseawell replied:
I can add a screenshot for each Mapping_Id scenario 👍

vfalon commented on help/destinations/catalog/advertising/magnite-batch.md:
```diff
@@ -161,19 +164,19 @@ On the "Configure a filename and export schedule for each audience" screen, you
 > A Mapping ID or "NONE" is required for this destination.
 >
-> A Mapping ID should be provided when an audience has a pre-existing Segment ID previously known to Magnite Streaming. Otherwise, "NONE" should be used as the Mapping ID.
+> A Mapping ID should be provided when an audience has a pre-existing Segment ID previously known to [!DNL Magnite]. Otherwise, "NONE" should be used as the Mapping ID.
```
Avoid using the passive voice. Talk directly to the reader:

"Provide a Mapping ID when your audience has a pre-existing Segment ID previously known to [!DNL Magnite]. Otherwise, enter "NONE" as the Mapping ID."

jakeseawell replied:
Sounds good 👍 I will update the wording.

vfalon reviewed (changes requested):
@jakeseawell please see my comments. Let's add some more clarity before rolling this update out.

jakeseawell committed:
update-documentation

 - Updated magnite-batch.md and magnite-streaming.md based on code-review comments/requests
 - Added 2 new screenshots for MappingId examples (batch)

jakeseawell commented:
@vfalon Thank you for the review. I have pushed up the requested changes to the PR. Can you please re-review, when you get a chance? Thanks!

vfalon commented on help/destinations/catalog/advertising/magnite-batch.md:
```diff
@@ -161,19 +164,19 @@ On the "Configure a filename and export schedule for each audience" screen, you
+> Provide a Mapping ID when your audience has a pre-existing Segment ID previously known to [!DNL Magnite]. Otherwise, enter "NONE" as the Mapping ID.
 >
-> When configuring the filename for each audience, please include the Mapping ID via the "Custom Text" field to add. The Mapping ID will be appended as: `{previous_filename}\_\[MAPPING_ID\].` If this audience is new to Magnite Streaming, and you will not be providing a Mapping ID, "NONE" should be entered into the "Custom Text" field. The new filename in this case should be: `{previous_filename}\ ...
+> When configuring the filename for each audience, please include the Mapping ID via the "Custom Text" field to add. The Mapping ID will be appended as: `{previous_filename}\_\[MAPPING_ID\].` If this audience is new to [!DNL Magnite], and you will not be providing a Mapping ID, "NONE" should be entered into the "Custom Text" field. The new filename in this case should be: `{previous_filename}\_\[ ...
```
please space out the images. One per row.

vfalon reviewed (changes requested):
@jakeseawell A couple more comments to address.

jakeseawell committed:
update-documentation

 - updated Adobe CDP -> Adobe Real-Time CDP
 - updated dropdown -> 'Select target field' popout window
 - renamed asset drowdown -> popout
 - Spaced out screenshots/assest to their own lines

jakeseawell commented:
@vfalon Added those updates. Thanks!

jakeseawell committed:
update-documentation

 - Added note on both docs about the magnite_deviceId_ fields being visible, but deprecated
